shared/src/crypto.rs

```diff
@@ -57,7 +57,8 @@ pub(crate) struct CertificateCache{
     cn_to_serial: HashMap<ProxyId, Vec<Serial>>,
     update_trigger: mpsc::Sender<oneshot::Sender<Result<(),SamplyBeamError>>>,
     root_cert: Option<X509>, // Might not be available at initialization time
-    im_cert: Option<X509> // Might not be available at initialization time
+    im_cert: Option<X509>, // Might not be available at initialization time
+    broken_certs: HashMap<Serial, SamplyBeamError>
```

Comment:
Please rename to `invalid_certs`

Reply:
Fixed (differently) in https://github.com/samply/beam/pull/106/commits/ad19055c086e634b3559e19c948e6e50cdfc952b